tsconfig.adf.json

```diff
@@ -36,7 +36,6 @@
       "@alfresco/aca-shared/store": ["projects/aca-shared/store/src/public-api.ts"],
       "@alfresco/aca-shared/rules": ["projects/aca-shared/rules/src/public-api.ts"],
       "@alfresco/aca-content/ms-office": ["projects/aca-content/ms-office/src/public-api.ts"],
-      "@alfresco/aca-testing-shared": ["projects/aca-testing-shared/src/index.ts"],
```

Comment:
New library should be added here

Reply:
@alfresco/playwright-shared ... its present on line 24, so did not move that.

Reply:
Ah okay didn't saw that